src/test/fuzz/connman.cpp

```diff
@@ -98,9 +98,6 @@ FUZZ_TARGET_INIT(connman, initialize_connman)
             [&] {
                 (void)connman.GetNodeCount(fuzzed_data_provider.PickValueInArray({ConnectionDirection::None, ConnectionDirection::In, ConnectionDirection::Out, ConnectionDirection::Both}));
             },
-            [&] {
-                connman.MarkAddressGood(random_address);
```

Comment:
61aecae0dafcc4c050343ef9c3e0242a11614cd9

what is the point of keeping the unused variable?

Reply:
Done (also removed the unused random_service variable)